symengine/complex_double.h

```diff
@@ -67,50 +63,53 @@ class ComplexDouble : public Number {
     /*! Add ComplexDoubles
      * \param other of type Integer
      * */
-    RCP<const Number> addcomp(const Integer &other) const {
+    RCP<const Number> addcomp(const Integer &other) const
+    {
         return make_rcp<const ComplexDouble>(i + mp_get_d(other.i));
     }
 
```

Comment:
All single line functions above were written as
```
type function { single-line-function-here }
```
Why wasn't that followed here? Max character limit?